### DISCUSSION TITLE
Exit preview early

### DISCUSSION THREAD

**Discussion by vforget** [quarto-dev/quarto-cli#6144] [Q&A]
### Description

I am using Quarto and Python. I want to exit preview/render early. If using R I would normally:

```
knitr::knit_exit()
```

What is the equivalent for Python?

**Comment by cscheid** [comment]
As far as I can tell, it doesn't exist. I would recommend you reorganize your code to include a boolean variable that you test for at the beginning of the cells you want to control.

**Comment by mcanouil** [comment]
FYI: <https://stackoverflow.com/questions/24005221/ipython-notebook-early-exit-from-cell>
